content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+In the Delivery lens, you can see CI pipeline and static analysis results associated to your services.
+
+By default, your service is associated to [CI pipeline(s)][4] via repository URL. 
+You can edit the pipelines associated with each service by modifying the ci-pipeline-fingerprints field in your [service metadata][5].
+
+To add or remove a pipeline associated to your service, first click on Edit Metadata in your service page and go to Software Delivery. Please note that this is only available for [Service Catalog schema v2.2][7]
```

Comment:
`Edit Metadata`

Nit but please add a period at the end of the last sentence :)